src/components/modal/report/report-options.jsx

```diff
@@ -80,7 +80,21 @@ const REPORT_OPTIONS = [
     {
         value: '6',
         label: {id: 'general.other'},
-        prompt: <FormattedMessage id="report.promptGuidelines" />,
+        prompt: (
+            <FormattedMessage
+                id={`report.promptGuidelines`}
```

Comment:
Prefer just `="report.promptGuidelines"`, the curly brackets + quotes are not needed here